src/main/webapp/app/pages/curation/review/ReviewPage.tsx

```diff
@@ -86,28 +90,45 @@ const ReviewPage: React.FunctionComponent<IReviewPageProps> = (props: IReviewPag
     }
   }, [geneData, reviewUuids, props.drugList]);
 
-  const acceptAllChangesFromEditors = (editors: string[]) => {
+  const acceptAllChangesFromEditors = async (editors: string[]) => {
     let reviewLevels = [] as ReviewLevel[];
     for (const editor of editors) {
       reviewLevels = reviewLevels.concat(editorReviewMap.getReviewsByEditor(editor));
     }
-    props.acceptReviewChangeHandler(hugoSymbol, reviewLevels, isGermline);
+    try {
+      setIsAcceptingAll(true);
+      await props.acceptReviewChangeHandler(hugoSymbol, reviewLevels, isGermline, true);
+      fetchFirebaseData();
+    } catch (error) {
+      notifyError(error);
+    } finally {
+      setIsAcceptingAll(false);
+      setEditorsToAcceptChangesFrom([]);
+    }
   };
 
-  const markRootCollapsibleAsPending = (reviewLevelId: string) => {
-    const newReviewChildren = reviewChildren.map(c => {
-      if (c.id === reviewLevelId) {
-        c.hideLevel = true;
-      }
-      return c;
-    });
+  const deleteCollapsible = (reviewLevelId: string, isPending = false) => {
```

Comment:
Maybe either separate into two methods or add a comment so it's clear that this method is called twice

Reply:
Separated logic into `deleteCollapsible` and `hideCollapsible`.